src/interaction/InteractionManager.js

```diff
@@ -414,12 +437,52 @@ InteractionManager.prototype.processMouseDown = function ( displayObject, hit )
 
     if(hit)
     {
-        displayObject[ isRightButton ? '_isRightDown' : '_isLeftDown' ] = true;
-        this.dispatchEvent( displayObject, isRightButton ? 'rightdown' : 'mousedown', this.eventData );
+        // only when the double click callback is defined we should perform
+        // the logic for checking double clicks
+        if(displayObject.dblclick || displayObject._listeners.dblclick)
+        {
+            this._clickCount++;
+
+            if(this._clickCount === 1)
+            {
+                var self = this;
+
+                this._dblClickTimer = setTimeout(function() {
+                    displayObject[ isRightButton ? '_isRightDown' : '_isLeftDown' ] = true;
+                    self.dispatchEvent( displayObject, isRightButton ? 'rightdown' : 'mousedown', self.eventData );
+                    self._clickCount = 0;
+                }, displayObject.dblClickDelay);
+            }
+            else
+            {
+                clearTimeout(this._dblClickTimer);
+                this.dispatchEvent( displayObject, 'dblclick', this.eventData );
+                this._clickCount = 0;
```

Comment:
This is not true as for 3 clicks we set the clickCount to 0. This violates the open-close principle if we want to ever extend the module.